griptape/tasks/base_text_input_task.py

```diff
@@ -15,7 +15,9 @@
 class BaseTextInputTask(RuleMixin, BaseTask, ABC):
     DEFAULT_INPUT_TEMPLATE = "{{ args[0] }}"
 
-    _input: str | TextArtifact | Callable[[BaseTask], TextArtifact] = field(default=DEFAULT_INPUT_TEMPLATE)
+    _input: str | TextArtifact | Callable[[BaseTask], TextArtifact] = field(
```

Comment:
No `Union` here? :)

Reply:
The use of `Union` is really only required if the field is marked as serializable. Related to [this comment](https://github.com/griptape-ai/griptape/pull/591/files/d2b66fa7a74ffef02151c085fd752d4effa70e08#r1484892627).